### DISCUSSION TITLE
2024-06-14(금)

### DISCUSSION THREAD

**Discussion by Seoin02** [Together-3team/petFrontend#144] [데일리스크럼]
아래 내용을 복사해 작성해주세요.
```
### 완료한 일

### 해야할 일

### 어려운 일
```

**Comment by q45402sk** [comment]
### 완료한 일

### 해야할 일
상품 상세 페이지-비슷한 상품 캐러셀 적용
상품 상세 페이지- 하단 모달
배송지 

### 어려운 일

**Comment by minyoung0503** [comment]
### 완료한 일

### 해야할 일
my -> 리뷰 상세보기 페이지 UI

### 어려운 일

**Comment by seolsis** [comment]
### 완료한 일
- 장바구니 api 연동 (상품목록 get, 상품 delete, 변경된 수량 put)
- api 파일 분리
- header 컴포넌트 적용

### 해야할 일
- toast
- 결제 실패 UI 구현 (drawer)
### 어려운 일

**Comment by Seoin02** [comment]
### 완료한 일
- axois intercepter 수정&미들웨어 설정
- 주문 상세 페이지 UI

### 해야할 일
- 인증 코드리뷰 반영
- 맞춤 상품, 주문 상세 페이지 컴포넌트 분리
- 온보딩, 프로필 API 연결

### 어려운 일

**Comment by devwqc** [comment]
### 완료한 일
- 없음

### 해야할 일
- BottomSheet를 포함한 모달 오늘 완료 예정, 금 ~ 일 모든 페이지 API까지 완료 예정 이였으나...
- BottomSheet를 오늘까지 해보고 안되면 포기하겠습니다. 페이지 작업은 토 ~ 월 예정입니다!

### 어려운 일
- 파이팅팅팅!!!👍💯
